tests/core/snap-auto-import-asserts/task.yaml

```diff
@@ -1,8 +1,5 @@
 summary: Check that `snap auto-import` works as expected
 
-# TODO:UC20: enable for UC20
-systems: [ubuntu-core-1*-64]
```

Comment:
I need to check if it works on arm device.

Reply:
I'll remove these for now and just run on all systems except arm with:
```yaml
systems: [ubuntu-core-*-64]
```